src/class/service/events.service.ts

```diff
@@ -0,0 +1,143 @@
+// Import Third-party Dependencies
+import { RedisAdapter } from "@myunisoft/redis";
+
+// Import Internal Dependencies
+import { IncomerStore } from "../store/incomer.class.js";
+import { Transaction, TransactionStore } from "../store/transaction.class.js";
+import type { RegisteredIncomer } from "../../types/index.js";
+import { EventEmitter } from "node:stream";
```

Comment:
```suggestion
// Import Node.js Dependencies
import { EventEmitter } from "node:stream";

// Import Third-party Dependencies
import { RedisAdapter } from "@myunisoft/redis";

// Import Internal Dependencies
import { IncomerStore } from "../store/incomer.class.js";
import { Transaction, TransactionStore } from "../store/transaction.class.js";
import type { RegisteredIncomer } from "../../types/index.js";
```
Il faudra update la config eslint on gère ça maintenant :p

Reply:
Yes je le ferai dans une PR de depup